news/2018-08-17-the-qat-gazette-july-2018.md

```diff
@@ -0,0 +1,97 @@
+
+### Beatmap Nominator Re-Evaluations
+
+Throughout July, we conducted a thorough review of activity, proficiency and behaviour for all current members of the Beatmap Nominators, in an attempt to ensure the quality of their collective work, and to provide help and guidance on how to improve where necessary. Depending on each Nominator's performance in these three areas, varying degrees of action were taken, ranging from promotion / retention of full Beatmap Nominator status for sufficient performance and above, to probationary Beatmap Nominator status for insufficient performance.
+
+Probationary status currently operates under a “one warning” system — this means that Nominators who were currently on, or had been previously been placed on probation for any given reason were dismissed from the team if found to be performing below expectation for a second time. Regardless of the reason for dismissal, we thank each Nominator for their service to the mapping and modding community and invite them to return in a later application cycle should our concerns be satisfactorily addressed.
+
+Feedback was provided by Quality Assurance Team members in each Nominator's respective mode(s), explaining the reasons for the decision taken, as well as indicating any areas for potential improvement if applicable.
+
+As a result of the Beatmap Nominator Re-Evaluations, 49/95 Nominators were retained as, or promoted to, full member status, 34/95 were retained or placed on probation, and 12/95 were removed from the group. Future evaluations will take place in conjunction with new Beatmap Nominator application phases on a six-weekly basis.
+
+### New Beatmap Nominator Promotions
+
+Whilst our existing Beatmap Nominators were being reviewed, we were pleased to welcome a wave of some new and returning circle inspectors to the team. Please offer warm welcome and congratulations to the following on their admission into the Beatmap Nominators:
+
+[BOUYAAA](https://osu.ppy.sh/users/405449), [Cris-](https://osu.ppy.sh/users/6175280), [frogyfro](https://osu.ppy.sh/users/7010859), [Gottagof4st](https://osu.ppy.sh/users/6122935), [IceKalt](https://osu.ppy.sh/users/5410645), [MrSergio](https://osu.ppy.sh/users/2581696), [pimpG](https://osu.ppy.sh/users/328615), [riffy](https://osu.ppy.sh/users/597957), [Rizen](https://osu.ppy.sh/users/3056732), [Seto Kousuke](https://osu.ppy.sh/users/2857314), [UndeadCapulet](https://osu.ppy.sh/users/2523533), [Yauxo](https://osu.ppy.sh/users/898306) and [Yugu](https://osu.ppy.sh/users/3161834)!
+
+May their efforts bring you many Ranked-beatmap-sized bundles of happiness over the coming weeks, months, and maybe even years!
+
+### Beatmap Nominator Retirements
```

Comment:
Only retirements?, what happens with the recent purge of BNs that werent added into the previous gazette?

Reply:
I think the purged bns mentioned in the previous section aren't included since it's basically highlighting "these people weren't good enough", whereas normally removals are less obvious to reasons.

Reply:
Sinnoh is correct, Nominators who are *dismissed* are not mentioned so as not to draw unneeded negative consequence for either party, and we typically only announce BNs who voluntarily *retire* in the Gazette.